### DISCUSSION TITLE
Might have found a bug in theming?

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2463] [Q&A]
This is a weird one. 

In lxqt-runner.qss files in any given theme, the directives in #commandList::item:$PSEUDOSTATE are not respected, and I can't find any other way to affect them. It's not just my themes either; I tried modifying Dark, Silver, etc and got the same results.

Relevant code snippet from a theme I'm working on:
```
/* Search results list */
#commandList {
        background:                     #ffffff;
        color:                          #000000;

        selection-color:                #ffffff;
        selection-background-color:     #606080;

        alternate-background-color:     #ffffff;

        border:                         1px solid #b3b3b3;
}

#commandList::item,
#commandList::item:alternate {
        color:                          #000000;
        background:                     #ffffff;
}

#commandList::item:hover,
#commandList::item:selected,
#commandList::item:selected:hover {
/* Pure red and blue for testing only, will be #606080 in production */
        color:                          #ffffff;
        background:                     #ff0000;

        selection-color:                #ffffff;
        selection-background-color:     #00ffff;
}
```

The expected result is that I'd get white text and either a blue or red background depending on the pseudostate of the list item. What actually happens is that the `background` directive is respected, _but none of `color`, `selection-color`, or `selection-background-color` are._ Further, the `selection-background-color` directive in the last stanza overrides the one in the `#commandList` section, though this might be working as intended...?

I tried writing each item:pseudostate pair as its own redundant block of QSS too, and no difference. Does anyone know what's going on here? This is bizarre...

**Comment by tsujan** [comment]
It's somehow complicated:

First, `selection-background-color` is a property of `QListView` ("#commandList"); it has no effect under `QListView::item`.

Second, *in the case of `QListView`*, `selection-background-color` is not only overriden by the background color of `QListView::item` (if existing) but also by the current Qt widget style — unless it's a gradient and the current Qt widget style doesn't use a gradient for drawing view-items (some styles do). So, it isn't reliable with `QListView`.

Third, if I correctly remember, Qt's Stylesheet style had a problem and didn't apply the text color of `QListView::item:hover`. If that's the case, the background color of `QListView::item:hover` needs to have enough contrast with the text color(s) of items.
